demos/voting/schemas/schemas.js

```diff
@@ -45,7 +45,8 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                         "lang": ["lang-en-US", "lang-fr-FR", "lang-es-MX", "lang-de-DE", "lang-nl-NL", "lang-sv-SE"],
                         "tooltip": ["lang-en-US-tooltip", "lang-fr-FR-tooltip", "lang-es-MX-tooltip", "lang-de-DE-tooltip", "lang-nl-NL-tooltip", "lang-sv-SE-tooltip"],
                         "tooltipAtSelect": ["lang-en-US-tooltipAtSelect", "lang-fr-FR-tooltipAtSelect", "lang-es-MX-tooltipAtSelect", "lang-de-DE-tooltipAtSelect", "lang-nl-NL-tooltipAtSelect", "lang-sv-SE-tooltipAtSelect"]
-                    }
+                    },
+                    "gradeNames": ["gpii.firstDiscovery.panel.lang.prefEditorConnection"]
```

Comment:
it might be easier to find if the gradeNames are higher up, like they are in a component definition. What do you think?